diff --git a/fontinst/MinionPro/make-oml-fullfamily-opticals.tex b/fontinst/MinionPro/make-oml-fullfamily-opticals.tex
index 8745c48..3768373 100644
--- a/fontinst/MinionPro/make-oml-fullfamily-opticals.tex
+++ b/fontinst/MinionPro/make-oml-fullfamily-opticals.tex
@@ -17,12 +17,8 @@
 \installfonts
   \installopt{Capt}{7}{Mixed}{l1-}
   \installopt{Capt}{7}{Mixed}{}
-  \installopt{Subh}{12}{Mixed}{}
-  \installopt{Disp}{12}{Mixed}{}
   \installopt{Capt}{7}{French}{l1-}
   \installopt{Capt}{7}{French}{}
-  \installopt{Subh}{12}{French}{}
-  \installopt{Disp}{12}{French}{}
 \endinstallfonts
 
 \bye
diff --git a/scripts/makeall b/scripts/makeall
index ec5c763..953f98e 100755
--- a/scripts/makeall
+++ b/scripts/makeall
@@ -51,9 +51,4 @@ then
   exit 1
 fi
 
-if $math; then
-    (cd fontinst; ../scripts/makeint $ffamily; ../scripts/makeoml $ffamily)
-fi
-
-(cd tex; latex $ffamily.ins; latex otfontdef.ins)
-
+(cd fontinst; ../scripts/makeoml $ffamily)
diff --git a/scripts/makeoml b/scripts/makeoml
index 9920990..89dec28 100755
--- a/scripts/makeoml
+++ b/scripts/makeoml
@@ -55,26 +55,7 @@ run_fontinst() {
   rm "$font"*.mtx MnSymbol*mtx
 }
 
-run_fontinst smallfamily Regular It Bold BoldIt
-
-if [ -e "../tfm/$font-Semibold-tosf-oml.tfm" ]
-then
-  run_fontinst medfamily Semibold SemiboldIt
-fi
-
-if [ -e "../tfm/$font-Medium-tosf-oml.tfm" ]
-then
-  run_fontinst fullfamily Medium MediumIt
-fi
-
-if [ -e "../tfm/$font-Capt-tosf-oml.tfm" ]
-then
-  run_fontinst medfamily-opticals Capt Subh Disp ItCapt ItSubh ItDisp SemiboldCapt SemiboldSubh SemiboldDisp SemiboldItCapt SemiboldItSubh SemiboldItDisp BoldCapt BoldSubh BoldDisp BoldItCapt BoldItSubh BoldItDisp
-fi
-
-if [ -e "../tfm/$font-MediumCapt-tosf-oml.tfm" ]
-then
-  run_fontinst fullfamily-opticals MediumCapt MediumSubh MediumDisp MediumItCapt MediumItSubh MediumItDisp
-fi
+  run_fontinst fullfamily Medium MediumIt
+  run_fontinst fullfamily-opticals MediumCapt MediumItCapt
 
 for v in *.vpl
diff --git a/scripts/maketfm b/scripts/maketfm
index a9c38fd..ba6cd72 100755
--- a/scripts/maketfm
+++ b/scripts/maketfm
@@ -73,16 +73,6 @@ wide_spacing=false
 glyph_list=""
 base_font_list=""
 
-if ! otfinfo -f otf/$font-Regular.otf | grep -q smcp
-then 
-  create_smallcaps=false
-fi
-
-if ! otfinfo -f otf/$font-It.otf | grep -q Swash
-then 
-  create_swash=false
-fi
-
 for opt in "$@"
 do
   case "$opt" in
diff --git a/scripts/maketfm-MinionPro.cfg b/scripts/maketfm-MinionPro.cfg
index 0823b04..2830463 100755
--- a/scripts/maketfm-MinionPro.cfg
+++ b/scripts/maketfm-MinionPro.cfg
@@ -2,15 +2,7 @@
 
 # Determine version of the fonts.
 
-case $(otfinfo -v otf/$font-Regular.otf) in
-  *";PS 001.000;"*) version=1;;
-  *";PS 001.001;"*) version=2;;
-  *"Version 2.012;PS 002.000;"*) version=3;;
-  *"Version 2.015;PS 002.000;"*) version=3;;
-  *"Version 2.030;PS 002.000;"*) version=4;;
-  *"Version 2.068;PS 002.000;"*) version=4;;
-  *)                version=5;;
-esac
+version=3
 
 # Newer versions have different glyph names.
 
